src/main/java/com/sobytylnik/ProfileController.java

```diff
@@ -0,0 +1,45 @@
+package com.sobytylnik;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.Bean;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+import java.util.Optional;
+
+
+@RestController
+public class ProfileController {

```

Comment:
You can specify base url for controller with `@RequestMapping(value="/api/profiles")`. It will decrease code duplication in methods.
```
@RequestMapping(method = RequestMethod.GET)
public List<Profile> getAllProfiles() { ... }

@RequestMapping(value="/{id}", method = RequestMethod.GET)
public Optional<Profile> getProfileByID(@PathVariable("id") Long id) { ... }
```